web-bluetooth/watch-advertisements-async-await.html

```diff
@@ -0,0 +1,35 @@
+---
+feature_name: Web Bluetooth / Watch Advertisements (Async Await)
```

Comment:
I like these samples.  watchAdvertisements will likely ship and be available for a long duration before requestLEScan becomes available.  I suggest adding a watchAdvertisements sample similar to https://googlechrome.github.io/samples/web-bluetooth/scan.html which does not try to connect, but only reads out advertisement data.  There are use cases for monitoring advertisements that will become viable with only requestDevice then watchAdvertisements.

I feel that the 'watch and connect' samples here are also valuable, but should be renamed more clearly to indicate they are examples focused on reconnecting and not reading AD packet data.

Reply:
Fixed.